nndownload/nndownload.py

```diff
@@ -275,16 +274,16 @@ def parse_datetime_to_timestamp(value) -> int:
 dl_group.add_argument("-c", "--download-comments", action="store_true", dest="download_comments",
                       help="download video comments")
 dl_group.add_argument("--comments-limit", dest="comments_limit", metavar="N", type=int, default=COMMENTS_LIMIT_DEFAULT_N,
-                      help=f"number of comments to download (default: {COMMENTS_LIMIT_DEFAULT_N})")
```

Comment:
This value is being used per-thread, not overall. I'm not sure there's an easy way to slim down the overall count, so I'm just changing the help output